styles/lib/carnival/features/_exercises.scss

```diff
@@ -0,0 +1,72 @@
+@use 'lib/carnival/theme' as carnival;
+
+@mixin exercises($element, $number-width: null) {
+    #{$element} {
+        [data-type=exercise] {
+            margin-bottom: 0.7rem;
+            [data-type=problem] {
+                @include -problem-or-question($number-width: $number-width);
+            }
+            [data-type=solution] > .solution-title {
+                font-weight: bold;
+                font-size: 1.2rem;
+                line-height: 1.5rem;
+            }
+        }
+        [data-type=injected-exercise] {
+            margin-bottom: 0.7rem;
+            [data-type=exercise-question] {
+                @include -problem-or-question($number-width: $number-width);
+                > .os-problem-container {
+                    > div[data-type=exercise-context] {
+                        display: inline;
+                    }
+                    > span.os-divider {
+                        display: inline;
+                    }
+                }
+            }
+        }
+    }
+}
+
+// Private mixins
+@mixin -number($width: null) {
```

Comment:
Had to change to `null`. Without `if` statment and with `0` value property was not optional. It was appearing in notes exercises as well.

Reply:
ohhh I see, I misunderstood what you were trying to do. good to know that `null` as a default argument makes a property optional! thanks for figuring that out!